README_OS_X.md

```diff
@@ -230,8 +229,15 @@ Common arguments to control the build configuration are:
 
     `-DCMAKE_OSX_ARCHITECTURES='i386;x86_64'`
 
-  * Normally, homebrew installations of readline are detected automatically, and building with  
-    readline is only required if you plan to use SuperCollider from the terminal. To link to a  
+  * Normally, homebrew installations of libsndfile are detected automatically. To link to a
+    non-standard version of libsndfile, you can use. This is also necessary if for some
```

Comment:
"you can use." ... ?  

I think this sentence is supposed to be "you can use:" and then follow with the options and then the next sentence.